tests/build_arm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -x
+
+ALPINE_VER="3.8"
+DISTRO="balenalib/rpi-alpine:$ALPINE_VER"
+# Used for Radicale
+EDGE_DISTRO="balenalib/rpi-alpine:edge"
+# Used for webmails
+PHP="arm32v7/php:7.2-apache"
+
+# use qemu-*-static from docker container
+docker run --rm --privileged multiarch/qemu-user-static:register 
```

Comment:
As I understand correctly from your comment, the `qemu-user-static` binary is already present in the OS images you are pulling? Why does this need to be run?